docs/DeveloperGuide.md

```diff
@@ -1013,10 +1013,71 @@ testers are expected to do more *exploratory* testing.
       than the number of assets/liabilities/policies listed).
       Expected: Similar to previous.
 
+### 7.6 Clearing client list
+
+1. Clearing all Clients in the home window.
```

Comment:
think ur clients should be in lower case for all headers

Reply:
```suggestion
1. Clearing all clients in the home window.
```

Reply:
done